lib/mix/tasks/transport/seed.ex

```diff
@@ -0,0 +1,298 @@
+        license: "odc-odbl",
+        title: "Horaires th\u00e9oriques du r\u00e9seau de transport 'CD 31 ARC EN CIEL'",
+        anomalies: ["redirection"],
+        spatial: "Haute-Garonne",
+        logo: "https://www.data.gouv.fr/s/avatars/a3/4fddcd8a2441438a7d1ecb0dffa2dc-100.png",
+        slug: "horaires-theoriques-du-reseau-de-transport-cd-31-arc-en-ciel",
+        format: "GTFS"
+      },
+      %{
+        description: "Horaires th\u00e9oriques du r\u00e9seau 'Transports du Pays Voironnais' [PTU272 VOIRON (P\u00e9rim\u00e8tre de transport urbain)]",
+        download_uri: "http://www.itinisere.fr/fr/api-open-services/169/OpenData/download?fileName=VOIRON.GTFS.zip",
+        license: "odc-odbl",
+        title: "Horaires th\u00e9oriques du r\u00e9seau de transport 'Transports du Pays Voironnais'",
+        anomalies: [],
+        spatial: "Pays Voironnais",
+        logo: "https://www.data.gouv.fr/s/avatars/a3/4fddcd8a2441438a7d1ecb0dffa2dc-100.png",
+        slug: "horaires-theoriques-du-reseau-de-transport-transports-du-pays-voironnais",
+        format: "GTFS"
+      },
+      %{
+        description: "Horaires th\u00e9oriques du r\u00e9seau 'Transaltitude' [38 ISERE (D\u00e9partement)]",
+        license: "odc-odbl",
+        title: "Horaires th\u00e9oriques du r\u00e9seau de transport 'Transaltitude'",
+        anomalies: ["outofdate"],
+        spatial: "Is\u00e8re",
+        logo: "https://www.data.gouv.fr/s/avatars/a3/4fddcd8a2441438a7d1ecb0dffa2dc-100.png",
+        slug: "horaires-theoriques-du-reseau-de-transport-transaltitude",
+        format: "GTFS"
+      },
+      %{
+        description: "Horaires th\u00e9oriques du r\u00e9seau 'Tougo' [PTU338 GRESIVAUDAN (P\u00e9rim\u00e8tre de transport urbain)]",
+        download_uri: "http://www.itinisere.fr/fr/api-open-services/169/OpenData/download?fileName=GRESIV.GTFS.zip",
+        license: "odc-odbl",
+        title: "Horaires th\u00e9oriques du r\u00e9seau de transport 'Tougo'",
+        anomalies: [],
+        spatial: "Communaut\u00e9 de Communes du Pays du Gr\u00e9sivaudan",
+        logo: "https://www.data.gouv.fr/s/avatars/a3/4fddcd8a2441438a7d1ecb0dffa2dc-100.png",
+        slug: "horaires-theoriques-du-reseau-de-transport-tougo",
+        format: "GTFS"
+      },
+      %{
+        description: "Horaires th\u00e9oriques du r\u00e9seau 'STAN' [PTU66 NANCY (P\u00e9rim\u00e8tre de transport urbain)]",
+        license: "fr-lo",
+        title: "Horaires th\u00e9oriques du r\u00e9seau de transport 'STAN'",
+        anomalies: [],
+        spatial: "Communaut\u00e9 Urbaine de Nancy",
+        logo: "https://www.data.gouv.fr/s/avatars/a3/4fddcd8a2441438a7d1ecb0dffa2dc-100.png",
+        slug: "horaires-theoriques-du-reseau-de-transport-stan",
+        format: "GTFS"
+      }
+    ]
```

Comment:
wouldn't it be more readable to just let this data in a json file and read it?

Reply:
For seeding the database? No...

Reply:
But we can do so if does make it more readable for you.

Reply:
It's just that a ex file doesn't seem a good place to store data to me.
I would prefer a json, or another file format that is used to store data.

Reply:
Ok